chain-core/src/init/address.rs

```diff
@@ -135,6 +145,25 @@ impl RedeemAddress {
     }
 }
 
+impl CroAddress<RedeemAddress> for RedeemAddress {
+    fn to_cro(&self) -> Result<String, ()> {
+        let checked_data: Vec<u5> = self.0.to_vec().to_base32();
+        Bech32::new("crms".into(), checked_data)
+            .map_err(|_e| ())
```

Comment:
this throws away errors: https://docs.rs/bech32/0.6.0/bech32/enum.Error.html
for encoding, I guess there would only be an error if we made a mistake in the human-readable part? I guess there could be a compile-time assertion and this could be safely unwrapped?

Reply:
broken checksum happens often.
cannot be unwrapped or expected

Reply:
agree